src/backend/allocator/gbm.rs

```diff
@@ -93,6 +95,20 @@ impl GbmBuffer {
     ///
     /// Gbm might otherwise give us the underlying or a non-sensical modifier,
     /// which can fail in various other apis.
+    #[cfg(feature = "backend_drm")]
+    pub fn from_bo(bo: BufferObject<()>, implicit: bool) -> Self {
+        let drm_node = DrmNode::from_file(bo.device_fd()).ok();
+        Self::from_bo_with_node(bo, implicit, drm_node)
+    }
```

Comment:
I do not like all those conditional compilation around `backend_drm` here. Do we really want to do this or should we just enable `backend_drm` for `backend_gbm` implicitely?

Reply:
yeah, I don't think gbm without drm is that useful.

Reply:
done, `backend_gbm` now always enabled `backend_drm`